### PR TITLE
Capacitor_THT: Update and regenerate models

### PR DESCRIPTION
**Pull request by antoniovazquezblanco**
I have found that the capacitors in the repo are outdated compared with the latest scripts due to the fact that https://github.com/KiCad/kicad-footprints/pull/550 was not merged.

Updated the script to split capacitors in a slightly better way.

### CODE REVIEW

File: scripts/Capacitors_THT/make_Capacitors_THT.py [hunk 1]
```diff
@@ -181,7 +181,7 @@
         makeResistorRadial(seriesname=seriesname, rm=rm, w=w, h=d, ddrill=ddrill, R_POW=R_POW,
                                     type=type, w2=w2, x_3d=[0, 0, 0], s_3d=[1, 1, 1], has3d=1,
                                     specialfpname="", add_description=add_description, name_additions=name_additions,
-                                    specialtags=specialtags, classname="C", lib_name="${KISYS3DMOD}/Capacitor_THT_Rectangular", script3d=scr, height3d=h3d)
+                                    specialtags=specialtags, classname="C", lib_name="Capacitor_Rect_THT", script3d=scr, height3d=h3d)
```
Comment (poeschlr) [inline scripts/Capacitors_THT/make_Capacitors_THT.py]: I think this should be Capacitor_THT.

File: scripts/Capacitors_THT/make_Capacitors_THT.py [hunk 2]
```diff
@@ -208,7 +208,7 @@
         makeResistorRadial(seriesname=seriesname, rm=rm, rm2=rm2, w=w, h=d, ddrill=ddrill, R_POW=R_POW,
                                     type=type, w2=w2, x_3d=[0, 0, 0], s_3d=[1, 1, 1], has3d=1,
                                     specialfpname="", add_description=add_description, name_additions=name_additions,
-                                    specialtags=specialtags, classname="C", lib_name="${KISYS3DMOD}/Capacitor_THT_Rectangular", script3d=scr, height3d=10)
+                                    specialtags=specialtags, classname="C", lib_name="Capacitor_Rect_THT", script3d=scr, height3d=10)
```
Comment (poeschlr) [inline scripts/Capacitors_THT/make_Capacitors_THT.py]: Also Capacitor_THT

File: scripts/Capacitors_THT/make_Capacitors_THT.py (hunk 1 above)
Comment (antoniovazquezblanco) [reply scripts/Capacitors_THT/make_Capacitors_THT.py]: Given that there are so many rectangular capacitors, I was voting on splitting them into a separate library. Do you think that is a bad idea? http://kicad-pcb.org/libraries/klc/G1.2/

Comment (antoniovazquezblanco) [reply scripts/Capacitors_THT/make_Capacitors_THT.py]: https://github.com/KiCad/kicad-footprints/pull/1715

**Comment by antoniovazquezblanco** [comment]
I've made the requested changes

**Comment by antoniovazquezblanco** [comment]
Capacitors will no longer be split, but models need to be regenerated and updated

**Pull request closed without merging**